tests/test_door_control.py

```diff
@@ -0,0 +1,330 @@
+"""Test Axis Door Control API.
+
+pytest --cov-report term-missing --cov=axis.light_control tests/test_light_control.py
+"""
+
+import json
+import pytest
+
+import respx
+
+from axis.door_control import DoorControl
+from .conftest import HOST
+
+
+@pytest.fixture
+def door_control(axis_device) -> DoorControl:
+    """Returns the door_control mock object."""
+    return DoorControl(axis_device.vapix.request)
+
+
+@respx.mock
+async def test_update(door_control):
```

Comment:
All previous tests have a @pytest.mark.asyncio descriptor, if you can pin point what dependency you have in your system that rids that dependency please add it to this PR so I can remove the descriptor.